[EFR-02]Enterprise Feature Request - False Positive Triaging

ajinabraham opened the pull request:
<!-- Thank you for your contribution to MobSF! -->

### Describe the Pull Request

```
Enterprise Feature Request - False positive suppression support
```
Coverage

- [x] Android - Code, Manifest
- [x] iOS - Code 
- [x] iOS Binary
- [x] Support other params that contribute to score
- [x] API + Docs
- [x] Support for Score/PDF/JSON etc.
- [ ] Tests

[Fixes](https://github.com/MobSF/Mobile-Security-Framework-MobSF/pull/2000/commits/bbf0c0d03ea45e5ee3f1f3f0f972b68416a38580) https://github.com/MobSF/Mobile-Security-Framework-MobSF/issues/2043

### Checklist for PR

- [x] Run MobSF unit tests and lint `tox -e lint,test`
- [x] Tested Working on Linux, Mac, Windows, and Docker
- [ ] Add unit test for any new Web API (Refer: `StaticAnalyzer/tests.py`)
- [ ] Make sure tests are passing on your PR [![MobSF tests](https://github.com/MobSF/Mobile-Security-Framework-MobSF/workflows/MobSF%20tests/badge.svg?branch=EFR-fp)](https://github.com/MobSF/Mobile-Security-Framework-MobSF/actions)

### Additional Comments (if any)

```
DESCRIBE HERE
```

matandobr commented on mobsf/static/others/js/suppression.js:
```diff
@@ -0,0 +1,153 @@
+}
+
+function list_suppressions(){
+  action(document.location.origin + '/list_suppressions/', { checksum: hash }, function(json) {
```
Don't you prefer using url names? this is prone to some errors I guess

ajinabraham replied:
I can update that.

matandobr reviewed (changes requested):
**
![image](https://user-images.githubusercontent.com/6709304/193273542-1308c6ae-b637-4034-b5d6-bf3b8b8a7d81.png)
**
This happened after I deleted a suppression by rule ID

Also I think the search is a bit weird because you are not splitting the lines, so each search will eaither return 0, 1, or 2 rows... not so helping I guess
![image](https://user-images.githubusercontent.com/6709304/193274468-b55b8753-2a64-43a8-bd40-5bd0aa599aff.png)

ajinabraham commented on mobsf/MobSF/settings.py:
```diff
@@ -149,7 +149,7 @@
 """
 # ===============================================
-
+DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
```
This is something needed in newer version of Django to support auto incrementing primary key in new modals

https://www.geeksforgeeks.org/autofield-django-models/

ajinabraham commented on mobsf/MobSF/init.py:
```diff
@@ -10,7 +10,7 @@
+VERSION = '3.6.0'
 BANNER = """
   __  __       _    ____  _____       _____  ____  
  |  \/  | ___ | |__/ ___||  ___|_   _|___ / | ___| 
```
Yea, sure

ajinabraham commented:
@matandobr Addressed some of the review comments.

> This happened after I deleted a suppression by rule ID

Can you reproduce this reliably/ share steps to reproduce? I cannot reproduce this at my end.
I know that there is an edge case where Data tables will fail if the table is not created properly in time.


> Also I think the search is a bit weird because you are not splitting the lines, so each search will either return 0, 1, or 2 rows... not so helping I guess

I am not sure I understand you. The search is a feature provided by Data tables itself.

matandobr reviewed (approved):
LGTM, great work 🎈🤙
